wodles/aws/aws_s3.py

```diff
@@ -1006,22 +1004,32 @@ def _event_should_be_skipped(event_):
 
                 self.send_event(event_msg)
 
+    @staticmethod
+    def _print_no_logs_to_process_message(bucket, aws_account_id, aws_region):
+        base_message = '+++ No logs to process in bucket:'
+        if aws_account_id is not None and aws_region is not None:
+            debug(f"{base_message} {aws_account_id}/{aws_region}", 1)
+        else:
+            debug(f"{base_message} {bucket}", 1)
+
     def iter_files_in_bucket(self, aws_account_id=None, aws_region=None):
+        if aws_account_id is None:
+            aws_account_id = self.aws_account_id
+
         try:
             bucket_files = self.client.list_objects_v2(**self.build_s3_filter_args(aws_account_id, aws_region))
             if self.reparse:
                 debug('++ Reparse mode enabled', 2)
 
             while True:
                 if 'Contents' not in bucket_files:
-                    base_message = '+++ No logs to process in bucket:'
-                    if aws_account_id is not None and aws_region is not None:
-                        debug(f"{base_message} {aws_account_id}/{aws_region}", 1)
-                    else:
-                        debug(f"{base_message} {self.bucket}", 1)
+                    self._print_not_logs_to_prcess_message(self.bucket, aws_account_id, aws_region)
```

Comment:
Typo in the method's call
```suggestion
                    self._print_not_logs_to_process_message(self.bucket, aws_account_id, aws_region)
```